core/components/tagger/elements/snippets/taggergetresourceswhere.snippet.php

```diff
@@ -30,6 +31,7 @@
 $tagField = $modx->getOption('tagField', $scriptProperties, 'alias');
 $likeComparison = (int) $modx->getOption('likeComparison', $scriptProperties, 0);
 $matchAll = (int) $modx->getOption('matchAll', $scriptProperties, 0);
+$errorOnInvalidTags = $modx->getOption('errorOnInvalidTags', $scriptProperties, false);
```

Comment:
You can't pass boolean as property from MODX. So I'd use something like:
```php
intval($modx->getOption('errorOnInvalidTags', $scriptProperties, 0)) === 1
```

Reply:
@theboxer I've updated this